custom-recipes/api-connect/recipe.json

```diff
@@ -45,7 +45,7 @@
         {
             "name": "endpoint_url",
             "label": "URL template",
-            "description": "https://{{subdomain}}.service.com/user/{{username}}/details",
+            "description": "https://{{my_variable}}.service.com/user/{{username}}/details",
```

Comment:
{{subdomain|my_variable}}

Reply:
`subdomain_variable` ? I'm just afraid than `|` will send the wrong message...

Reply:
https://{{subdomain}}.service.com/user/{{username}}/details or https://{{subdomain}}.service.com/user/{{my_variable}}/details